_posts/2023-09-05-use-opensearch-benchmark-with-randomized-query-terms.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title:  "Use OpenSearch Benchmark with Randomized Query Terms"
+authors:
+- handler
+date: 2023-09-05
+categories:
+  - technical-post
+meta_keywords: opensearch-benchmark
+meta_description: Learn how to add a randomized query set to an OpenSearch Benchmark workload
```

Comment:
Hi Jon - Please update the meta with the following if you agree with the edits: 

**meta_keywords:** OpenSearch benchmark, OpenSearch benchmark workload, OpenSearch cluster
**meta_description:** Learn how to add a randomized query set to an OpenSearch benchmark workload to read and build a test framework for data running in your cluster.